### PR TITLE
Zb/theme revamp

### PR DESCRIPTION
**Pull request by zbarnz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,5 +43,11 @@
   ],
   "nyc": {
     "include": "api/lib/**/*.js"
+  },
+  "dependencies": {
+    "@emotion/react": "^11.10.0",
+    "@emotion/styled": "^11.10.0",
+    "@mui/material": "^5.10.1",
+    "deprank": "^0.1.1"
```
Comment (thudsonbu) [inline package.json:51]: We should put these in the `www/package.json`.

File: www/.npmrc [hunk 2]
```diff
@@ -1 +1 @@
-package-lock=false
+package-lock=false
```
Comment (thudsonbu) [inline www/.npmrc:1]: ?

File: www/components/Card.js [hunk 3]
```diff
@@ -44,13 +44,14 @@ const Card = ({
...
     <div className={styles.container + ' ' + styles.editing}>
-      <Input onChange={( e ) => setName( e.target.value ) } value={name}/>
+      <Input onChange={( e ) => setName( e.target.value ) } value={name} placeholder='Title'/>
       <Input
         multiLine={true}
         rows='5'
         value={description}
         variant='outlined'
         onChange={( e ) => setDescription( e.target.value )}
+        placeholder='Description...'
```
Comment (thudsonbu) [inline www/components/Card.js:54]: 👍🏻

File: www/components/CardBoard.js [hunk 4]
```diff
@@ -107,7 +111,7 @@ const CardBoard = ( props ) => {
         <Button
           className={styles.add_button}
           onClick={() => createItem()}
-          variant='icon'
+          variant='iconInverted'
```
Comment (thudsonbu) [inline www/components/CardBoard.js:114]: Should we just use the MUI button?

File: www/components/Drawer.js [hunk 5]
```diff
@@ -44,7 +44,7 @@ const Drawer = () => {
           icon={<CancelOutlinedIcon style={{ color: 'var(--danger)' }} />}
         />
       </div>
-      <div
+      {/* <div
```
Comment (thudsonbu) [inline www/components/Drawer.js:47]: Lets just remove this instead of commenting it out.

File: www/store/features/drawer.js [hunk 6]
```diff
@@ -1,5 +1,5 @@
 const initialState = {
-  open: false
+  open: true
```
Comment (thudsonbu) [inline www/store/features/drawer.js:2]: Why would we want this to be open? I don't think you understand how this is supposed to work.

File: www/styles/Shared.module.css [hunk 7]
```diff
@@ -1,7 +1,6 @@
 .card {
   color: var(--agendaText);
-  background-color: var(--backgroundTwo);
-  border: 1px solid var(--agendaSecondary);
+  border: 1px solid var(--agenda);
```
Comment (thudsonbu) [inline www/styles/Shared.module.css:3]: Why did you name the colors this way?

File: www/styles/Shared.module.css [hunk 8]
```diff
@@ -21,20 +20,32 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0em 0em 5em 0em;
+  max-height: 95vh;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  resize: none;
+}
+
```
Comment (thudsonbu) [inline www/styles/Shared.module.css:40]: Extra places here.

File: www/styles/Shared.module.css (hunk 7 above)
Comment (zbarnz) [reply www/styles/Shared.module.css:3]: @zbarnz legacy systems support

Comment (zbarnz) [reply www/styles/Shared.module.css:3]: @thudsonbu

File: www/styles/Shared.module.css [hunk 9]
```diff
@@ -21,20 +20,32 @@
...
+
 /* use psuedoselector for scrollbar (not supported by all browsers) */
 .page::-webkit-scrollbar {
   width: .75em;
 }
 
 .page::-webkit-scrollbar-track {
-  background: var(--background);
+  background: var(--backgroundTwo);
```
Comment (thudsonbu) [inline www/styles/Shared.module.css:48]: What is "backgroundTwo"?

File: www/styles/Shared.module.css (hunk 8 above)
Comment (zbarnz) [reply www/styles/Shared.module.css:40]: removed

File: www/styles/components/Button.module.css [hunk 10]
```diff
@@ -41,6 +41,19 @@
...
+  color: var(--agendaThree);
+  min-width: 0px;
+  padding: 0.2em;
+  margin-top: 0.2em;
+}
+
+.icon_inverted:hover {
+  opacity: 50%;
+}
+
```
Comment (thudsonbu) [inline www/styles/components/Button.module.css:55]: Extra pacing here.

File: www/styles/Shared.module.css (hunk 9 above)
Comment (zbarnz) [reply www/styles/Shared.module.css:48]: ?

File: www/styles/Shared.module.css (hunk 8 above)
Comment (thudsonbu) [reply www/styles/Shared.module.css:40]: You should try to think more.

File: www/components/CardBoard.js (hunk 4 above)
Comment (zbarnz) [reply www/components/CardBoard.js:114]: lets create a seperate ticket

File: www/styles/components/Nav.module.css [hunk 11]
```diff
@@ -59,4 +62,9 @@
 
 .nav_button {
   margin-left: 1em;
+  color: white;
```
Comment (thudsonbu) [inline www/styles/components/Nav.module.css:65]: Why didn't you use a variable here?

File: www/styles/components/Nav.module.css [hunk 12]
```diff
@@ -59,4 +62,9 @@
...
+}
+
+.icon_button {
+  color: white;
```
Comment (thudsonbu) [inline www/styles/components/Nav.module.css:69]: Why didn't you use a variable here?

File: www/styles/globals.css [hunk 13]
```diff
@@ -16,6 +16,12 @@ body {
   overflow: hidden;
 }
 
+h3 {
```
Comment (thudsonbu) [inline www/styles/globals.css:19]: Do we want the extra margin on all `h3`s.

File: www/styles/pages/meeting/[id]/index.module.css [hunk 14]
```diff
@@ -1,3 +1,7 @@
 .meetingButton {
   margin-left: auto;
 }
+
+.section {
+  margin: 1.5em 0 1.5em 0;
+}
```
Comment (thudsonbu) [inline www/styles/pages/meeting/[id]/index.module.css:7]: There should be a newline here. Try to look at things before you open up a pull request.

File: www/styles/globals.css (hunk 13 above)
Comment (zbarnz) [reply www/styles/globals.css:19]: sorry that was a poor implementation of that style

File: www/styles/pages/meeting/[id]/index.module.css (hunk 14 above)
Comment (zbarnz) [reply www/styles/pages/meeting/[id]/index.module.css:7]: lets create a linting rule to add new lines if its such a big deal :) I have seen this in your pull requests as well so it seems like a common issue

**Comment by thudsonbu** [comment]
I think you need to take a second to look over what you wrote here and ask yourself if you are proud of this.

File: www/styles/Shared.module.css (hunk 7 above)
Comment (thudsonbu) [reply www/styles/Shared.module.css:3]: @thudsonbu 
@zbarnz 
@thudsonbu :brain"

**Review by thudsonbu — APPROVED** [review]

**Comment by thudsonbu** [comment]
Merge it!

**Pull request merged**